### PR TITLE
Message Store Features

### PR DESCRIPTION
**Pull request by jcope**
Adds 4 buttons to the control panel pertaining to message storage:
-Add (current message to the localdb storage)
-View (displayed crudely in an alert window)
-Export (to file, json format)
-Import (to the localdb)

### CODE REVIEW

File: browser/dasher/messageStore.js [hunk 1]
```diff
@@ -27,8 +28,12 @@ export default class MessageStore {
 
     viewMessageStore(){
       this._load_from_browser().catch(() => {}).then(messages => {
```
Comment (sjjhsjjh) [inline browser/dasher/messageStore.js:30]: Maybe add a comment near here that says what `messages` is. Looks like it is an array of arrays of string?

File: browser/dasher/messageStore.js [hunk 2]
```diff
@@ -27,8 +28,12 @@ export default class MessageStore {
...
-        //TODO - present these message in another place other than the console
-        console.log(messages);
+        //TODO - present these messages in a usable format
+        let output = [];
```
Comment (sjjhsjjh) [inline browser/dasher/messageStore.js:32]: Maybe use Array flat() to do this instead.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat

**Review by sjjhsjjh — CHANGES_REQUESTED** [review]

File: browser/dasher/messageStore.js (hunk 1 above)
Comment (jcope) [reply browser/dasher/messageStore.js:31]: Somewhat better, looks like a map was better for this case

**Comment by jcope** [comment]
If there are no further suggestions, I'd like to move forward and merge and close this into the main branch.

**Pull request merged**